pkg/Library/Polymath/Atoms/AskPolymath.js

```diff
@@ -13,6 +13,6 @@ async update({library, query}, state, {service, isDirty}) {
     return;
   }
   const result = await service('PolymathService', 'Ask', {library, query});
-  return {query, result, completion: result.completion};
+  return {query, result, completion: result?.completion ?? 'n/a'};
```

Comment:
Nit: as a rule, factor it out if there is logic.

Reply:
factored